src/tests/Register.test.js

```diff
@@ -106,4 +107,70 @@ it('blocks the user to register without password confirmation', async () => {
 
     expect(screen.queryByLabelText("response")).not.toBeInTheDocument();
     
-})
\ No newline at end of file
+})
+
+it('checks empty field warning', async () => {
+
+    render(<Register />)
+
+
+    fireEvent.change(screen.getByLabelText("Name :", { selector: "input" }), {
+        target: { value: '' },
+    })
+
+
+    fireEvent.change(screen.getByLabelText("Username :", { selector: "input" }), {
+        target: { value: '' },
+    })
+
+    fireEvent.change(screen.getByLabelText("Email :", { selector: "input" }), {
+        target: { value: '' },
+    })
+
+    fireEvent.change(screen.getByLabelText("Password :", { selector: "input" }), {
+        target: { value: '' },
+    })
+
```

Comment:
Add another fireEveent.change for `Confirm Password :` field here to prevent error